web/src/components/navigation/mainNav.js

```diff
@@ -4,11 +4,32 @@ import { Link } from 'gatsby';
 
 export default function MainNav({ navItems }) {
     return (
-        <ul className="flex gap-3 justify-end uppercase mt-16 mr-24">
-            {navItems.map((item) => {
+        <ul className="flex font-semibold gap-3 justify-end mt-16 mr-24 text-white uppercase">
+            {navItems.map((item, i) => {
                 return (
-                    <li key={item.link}>
-                        <Link to={`${item.link}`}>{item.name}</Link>
+                    <li key={item.link} className="group">
+                        <Link
+                            className="hover-list group-hover:block"
+                            to={`${item.link}`}
+                        >
+                            {item.name}
+                        </Link>
+                        <ul>
+                            {navItems[i].subnav
```

Comment:
I think you should be able to do `item.subnav` instead of passing in the `index` value

Reply:
@asasmith that did not work, it does not know what item.subnav is

Reply:
```
{item.subnav
    ? item.subnav.map((subitem) => {
        return (
            <li key={subitem.sublink}>
                <Link className="hover-list-link p-2 group-hover:block text-brand-1 bg-brand-5 hover:bg-brand-4"
                      to={`${subitem.subnav}`}
                >
                     {subitem.sublink}
                </Link>
            </li>
        );
   })
: ''}
```


@kcbreuil this works. Did you have something different?

Reply:
@asasmith No when I did item.subnav, it said subnav was undefined. Not sure why though.